build_projects/shared-build-targets-utils/Publishing/AzurePublisher.cs

```diff
@@ -219,14 +219,21 @@ public void ReleaseLeaseOnBlob(string blob, string leaseId)
             cloudBlob.ReleaseLeaseAsync(ac).Wait();
         }
 
-        public bool IsLatestSpecifiedVersion(string version)
+        public bool DoesBlobReferenceExist(string version)
```

Comment:
I should have been clearer--I was suggesting renaming the `version` parameter to `blob` to match the new name of the method.